lib/rust/ensogl/core/src/display/scene.rs

```diff
@@ -485,25 +487,36 @@ impl DomLayers {
         let front = DomScene::new(logger);
         let fullscreen_vis = DomScene::new(logger);
         let back = DomScene::new(logger);
+        let panel = DomScene::new(logger);
+
+        panel.dom.set_class_name("panel");
+        panel.dom.set_style_or_warn("pointer-events", "auto", logger);
+        panel.dom.set_style_or_warn("z-index", "0", logger);
+
+        back.dom.set_class_name("back");
+        back.dom.set_style_or_warn("pointer-events", "auto", logger);
+        back.dom.set_style_or_warn("z-index", "1", logger);
+
+        fullscreen_vis.dom.set_class_name("fullscreen_vis");
+        fullscreen_vis.dom.set_style_or_warn("z-index", "2", logger);
+
         canvas.set_style_or_warn("height", "100vh", logger);
         canvas.set_style_or_warn("width", "100vw", logger);
         canvas.set_style_or_warn("display", "block", logger);
         // Position must not be "static" to have z-index working.
         canvas.set_style_or_warn("position", "absolute", logger);
```

Comment:
Here is an interesting comment – I see that other layers do not have the attrib 'position' set to 'absolute'. I know this is not part of this PR, but its always worth checking. If this comment is invalid, please remove it. If its valid, then are other z-indexes not working?

Reply:
According to HTML specification, the comment is correct, though `position: absolute` is set in `DomScene::new`, so I removed both the comment and this line.